studygroups/models.py

```diff
@@ -568,3 +576,104 @@ def report_data(start_time, end_time, team=None):
         report['team'] = team
     return report
 
+def get_json_response(url):
+    response = requests.get(url)
+    return response.json()
```

Comment:
I think `response.json()` will throw `json.decoder.JSONDecodeError` if the response body doesn't contain valid json. This will happen with responses like 404